fv/host_port_test.go

```diff
@@ -145,7 +149,21 @@ var _ = infrastructure.DatastoreDescribe("with initialized Felix", []apiconfig.D
 			})
 
 			It("port should be reachable", func() {
-				Eventually(metricsPortReachable, "10s", "1s").Should(BeTrue())
+				//Eventually(metricsPortReachable, "10s", "1s").Should(BeTrue())
```

Comment:
The fact that this seems to work around suggests that `metricsPortReachable` has a bug and takes >10s to time out, if that was the case then an Eventually(... 10, 1) would only run metricsPortReachable once so it would be flaky. 

Looking at the code that `metricsPortReachable` calls, i don't see any timeouts specified so I bet it defaults to something large like 60s.

Reply:
I just made that change (incorrectly) and noticed https://github.com/projectcalico/felix/pull/1763/files#diff-c663090db8354a82f2b72fd73f1dc935R28. Looks like there is already a 1 second timeout on the http.Get.
That was a great thought though.